diagnostic.md

```diff
@@ -100,11 +137,25 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --include --request PATCH "http://localhost:3000/books/$ID" \
+  --header "Content-Type: application/json" \
+  --data $"{
+    \"donuts\": {
+      \"name\": \"Krüller\",
+    }
+  }"
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+const changeDonut = function (form) {
+  let data= getFormFields(form);
```

Comment:
read: " do not use `data = getFormFields(form)` instead write out the data object"